binding.gyp

```diff
@@ -2,17 +2,17 @@
   "targets": [
     {
       "target_name": "cpu_profiler",
-      "sources": [ "src/cpu-profiler.cc" ],
+      "sources": [ "bindings/cpu-profiler.cc" ],
```

Comment:
We discussed previously that it would be good to avoid any native code in the profiler agent, but these bindings just have to exist because there is no JS-side interface to the profiler right now, yes?

@nolanmar511

Reply:
I am migrating the code from js to ts at this point. Changing this to not use native code would be nice, but should be done independently of this PR.